File: src/main/java/com/woomulwoomul/woomulwoomulbackend/domain/user/custom/UserRoleCustomRepository.java
package com.woomulwoomul.woomulwoomulbackend.domain.user.custom;

import com.woomulwoomul.woomulwoomulbackend.domain.user.UserRole;

import java.util.List;

public interface UserRoleCustomRepository {

    List<UserRole> findAllFetchUser(Long userId);
}

File: src/main/kotlin/com/woomulwoomul/woomulwoomulbackend/domain/user/custom/UserRoleCustomRepository.kt
package com.woomulwoomul.woomulwoomulbackend.domain.user.custom

import com.woomulwoomul.woomulwoomulbackend.domain.user.UserRoleEntity

interface UserRoleCustomRepository {

    fun findAllFetchUser(userId: Long): List<UserRoleEntity>

}
